tools/wpt/testharnessreport.js

```diff
@@ -1,13 +1,16 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+import { writeAllSync } from "../../test_util/std/io/util.ts";
+
```

Comment:
This will fail WPT tests. These files are not really modules, they behave more like scripts. We can't import in here. You'll either need to vendor in the `writeAllSync`, or just use `Deno.writeSync` and assert that the written byte length is correct.

Reply:
I have implemented this.